options.js

```diff
@@ -0,0 +1,113 @@
+// Saves options to chrome.storage
+var preferences = null;
+function save_options() {
+  chrome.storage.sync.set(preferences, function() {
```

Comment:
that's awfully specific to Chrome. If simple storage is what you're after, then you could use [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) or [indexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API).

If browser sync is important, then you'll have to abstract it away and handle specific browsers.

Reply:
Thanks for pointing that out, `chrome.storage` seems to work fine on Firefox and Safari even though those browsers prefer `browser.*` apis instead of `chrome.*` in their respective docs, but yes, we should keep an eye on this

Related doc: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Build_a_cross_browser_extension#api_namespace